Packs/NetscoutAED/Integrations/NetscoutAED/NetscoutAED.yml

```diff
@@ -0,0 +1,1286 @@
+      description: Number of passed bytes.
+      type: Unknown
+    - contextPath: NetscoutAED.Protection_Group.description
+      description: Description of the protection group.
+      type: String
+    - contextPath: NetscoutAED.Protection_Group.name
+      description: Protection group name.
+      type: String
+    - contextPath: NetscoutAED.Protection_Group.packets_dropped
+      description: Number of dropped packets.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.packets_passed
+      description: Number of passed packets.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.pgid
+      description: Protection group identifier.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.pps_passed
+      description: Number of passed pps.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.pps_dropped
+      description: Number of dropped pps.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.prefixes
+      description: A comma-separated list of prefixes belonging to the protection
+        group.
+      type: Unknown
+    - contextPath: NetscoutAED.Protection_Group.profiling
+      description: A traffic profile capture for a protection group’s rate-based protection
+        settings is running (true) or not (false).
+      type: Boolean
+    - contextPath: NetscoutAED.Protection_Group.profiling_duration
+      description: The duration, in days, of an active traffic profile capture. A
+        0 indicates that profiling is not active.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.profiling_start
+      description: A UNIX epoch timestamp that indicates when a traffic profile capture
+        began. A 0 indicates that profiling was never started.
+      type: Date
+    - contextPath: NetscoutAED.Protection_Group.protection_level
+      description: The protection level (None = use the global protection level, low,
+        medium, high).
+      type: Unknown
+    - contextPath: NetscoutAED.Protection_Group.server_name
+      description: The protection group’s server name.
+      type: String
+    - contextPath: NetscoutAED.Protection_Group.server_type
+      description: The protection group’s server type.
+      type: Number
+    - contextPath: NetscoutAED.Protection_Group.time_created
+      description: The time when the protection group was created.
+      type: Date
+  - arguments:
+    - default: false
+      description: Protection group identifier.
+      isArray: false
+      name: pgid
+      required: false
+      secret: false
+    - default: false
+      description: Protection group name.
+      isArray: false
+      name: name
+      required: false
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Whether the protection group is active or not.
+      isArray: false
+      name: active
+      predefined:
+      - 'true'
+      - 'false'
```

Comment:
As best practice - it is best to use the 'argToBoolean' CSP method in the code to handle boolean arguments.
(instead of arg == 'true' for example).

Reply:
Done